source/meetings/2022/april/index.html.md

```diff
@@ -0,0 +1,82 @@
+---
+updated_by:
+  email: paolo.fabbri@lrug.org
+  name: Paolo Fabbri
+created_by:
+  email: paolo.fabbri@lrug.org
+  name: Paolo Fabbri
+category: meeting
+title: April 2022 Meeting
+published_at: 2022-04-04 21:30:00 +0000
+created_at: 2022-04-04 20:51:00 +0000
+status: Published
+---
+
+The April 2022 meeting of LRUG will be on *Monday the 11th of April*,
+from _6:20pm_ to _8:00pm_ (meeting starts at _6:30pm_).
+
+[Full registration details are given below](#apr22registration), but do
+note that **we will only be sending out the zoom meeting url to people who
+have registered, because we have limited places, so please do it.**
+
+## Agenda
+
+### Using React in a Ruby Project to Dynamically Generate PDF Documents
+
+[Panos Matsinopoulos](https://getlavanda.com) says:
+
+
+> Fjust needs to be a couple of sentences really, not a
+> full abstract. We use markdown so feel free to format it or embed links for
+> the writeup.
+> 
+> In the Ruby world, we traditionally address the PDF generation problem
+> using gems like [Prawn](https://github.com/prawnpdf/prawn) and [PDFKit](https://github.com/pdfkit/pdfkit)
+> or libraries like [whtmltopdf](https://wkhtmltopdf.org/).
+> 
+> Recently, in one of our Ruby on Rails projects in which we 
+> wanted to generate PDF documents for invoices, we decided to use
+> another programming language and technology: React and AWS Lambda.
+> 
+> In this talk, we will be covering how we did it, what were the challenges
+> and what pros and cons over the incumbent tools for Ruby.
+
+{::coverage year="2022" month="april" talk="using-react-in-a-ruby-project-to-dynamically-generate-p-d-f-documents" /}
```

Comment:
A bit of a nitpick, but is it desirable to spell `pdf` as `p-d-f`?

Reply:
I've converted the case of the sentence with sublime text and it does that